packages/@react-spectrum/dialog/docs/DialogTrigger.mdx

```diff
@@ -332,31 +324,34 @@ prop. See the props table [below](#props) for a full list of available placement
 
 The Popover Dialog's offset with respect to its anchor element can be adjusted using the `offset` and
 `crossOffset` props. The `offset` prop controls the spacing applied along the main axis between the element and its
-anchor element whereas the `crossOffset` prop handles the spacing applied along the cross axis. 
+anchor element whereas the `crossOffset` prop handles the spacing applied along the cross axis.
 
+Below is a popover offset by an additional 50px above the trigger.
 ```tsx example
 <DialogTrigger type="popover" placement="top" offset={50}>
   <ActionButton>Trigger</ActionButton>
   <Dialog>
-    <Heading>The Heading</Heading>
+    <Heading>Offset</Heading>
     <Divider />
     <Content>
       <Text>
-        This is a popover offset by an additional 50px above the trigger.
+        Offset by an additional 50px.
       </Text>
     </Content>
   </Dialog>
 </DialogTrigger>
-
+```
+Below is a popover offset by an additional 100px to the right of the trigger.
+```tsx example
 <DialogTrigger type="popover" placement="top" crossOffset={100}>
   <ActionButton>Trigger</ActionButton>
   <Dialog>
-    <Heading>The Heading</Heading>
+    <Heading>Cross offset</Heading>
     <Divider />
     <Content>
       <Text>
-        This is a popover offset by an additional 100px to the right of the trigger.
-      </Text>   
+        Offset by an additional 100px.
```

Comment:
given that the other one is called offset, this is confusing to have Cross offset in the heading but offset in the description